### ISSUE TITLE
The normal task

### ISSUE THREAD

**Issue by CyunXiong** [TUI-NICR/EMSANet#13]
I wonder whether the normal task is working, it is possible to visualize the normal task？

**Comment by danielS91** [comment]
Yes, the normal task is working - we did some experiments on Hypersim and NYUv2. However, for NYUv2, the ground truth is no longer available. Training on Hypersim should work.
Just have a look at [lib/nicr_mt_scene_analysis/task_helper/normal.py](https://github.com/TUI-NICR/nicr-multitask-scene-analysis/blob/6a618cf89fff716b8059c0d81a7ac207ef16d4a3/nicr_mt_scene_analysis/task_helper/normal.py)
